enter-chroot.sh

```diff
@@ -9,4 +9,17 @@ then
     exit 1
 fi
 
-sudo systemd-nspawn -M homeworld --bind $(pwd):/homeworld:norbind -u "$USER" -a -D "$HOMEWORLD_CHROOT" bash -c "cd /h/ && exec bash"
+ORIG_PWD="$(pwd)"
+cd "$(dirname "$0")"
+ORIG_REL="$(realpath --relative-to "$(pwd)/building" "${ORIG_PWD}")"
+if [[ "${ORIG_REL}" =~ \.\. ]]
```

Comment:
What is the purpose of this code?  If I understand, if you run the script from a subdirectory of `building` then it spanws the new shell in the corresponding subdirectory of the chroot.  While I understand that it might be useful to control the subdirectory of the chroot used, why is using the caller's pwd a good way to do so?

Reply:
Also, why is this code only present in the systemd-nspawn variant?

Reply:
It's a good way to do so because, if you have a terminal open to a subdirectory and want to run a build, you can easily launch a shell that points to the same context you've already browsed to.

This is a human-oriented feature, because it's annoying to cd into a directory, launch a chroot, and then cd into the same directory *again*. It smooths over the interface between the chroot and the outside world.

It is only present in the systemd-nspawn variant because ubuntu 14.04 cannot run `realpath --relative-to`, which makes it much harder to implement, and it's a human-oriented feature, so less useful for automated environments.